tests/test_faster_rcnn.py

```diff
@@ -0,0 +1,377 @@
+import argparse
+import os
+from unittest.mock import MagicMock, patch
+
+import numpy as np
+import pytest
+import torch
+import torchvision
+from PIL import Image
+from tensorboardX import SummaryWriter
+from yacs.config import CfgNode as CN
+
+from datasetinsights.data.dummy.dummy_object_detection import DummyDetection2D
+from datasetinsights.estimators import Estimator
+from datasetinsights.estimators.faster_rcnn import (
+    collate_fn,
+    create_dataloader,
+    create_dryrun_dataset,
+    create_optimizer_lrs,
+    create_sampler,
+    download_data,
+    get_transform,
+    load_data,
+)
+from datasetinsights.storage.checkpoint import create_checkpointer
+
+TRAIN = "train"
+VAL = "val"
+TEST = "test"
+
+label_mappings = {"1": "car", "2": "bike"}
+dummy_data = DummyDetection2D(
+    transform=get_transform(), label_mappings=label_mappings
+)
+
+
+@pytest.fixture
+def config():
+    parser = argparse.ArgumentParser(
```

Comment:
Why do you need `ArgumentParser` here? This test is not going to run with CLI arguments? Can this method simply return  `cfg`?

Reply:
How to initialize parser then?

Reply:
You can use [yacs](https://github.com/rbgirshick/yacs) to do the initialization. And you can put `config` in a `conftest.py` ([example](https://www.tutorialspoint.com/pytest/pytest_conftest_py.htm)).

Reply:
This has to be fixed. It's absolutely not required to have argparse stuff in unittests, as there are no way for these arguments to be passed during the test execution.